scripts/covid19indiaORG/import-config.txt

```diff
@@ -0,0 +1,11 @@
+import: {
+  import_name: "covid19indiaORG"
```

Comment:
Covid19India

Reply:
I used covid19indiaORG because there was some other covid19 India import, didn't want to confuse them. But, done!